solr/solrj/src/java/org/apache/solr/client/solrj/impl/HttpSolrClientBuilderBase.java

```diff
@@ -38,7 +37,7 @@ public abstract class HttpSolrClientBuilderBase<
   protected Set<String> urlParamNames;
   protected Integer maxConnectionsPerHost;
   protected ExecutorService executor;
-  protected CookieStore cookieStore;
+  protected boolean useHttp1_1 = Boolean.getBoolean("solr.http1");
```

Comment:
I like it but I can't help but notice that no other field here has a default that is defined here.  So _maybe_ this new setting should be initialized elsewhere?

Reply:
This was moved from `Http2SolrClient` so I left it as-is.  I initially did not see any value in supporting this in the new client.  I reconsidered after realizing the stream-cancelling bug was http/2-specific.